py-polars/polars/cfg.py

```diff
@@ -64,6 +64,14 @@ def set_tbl_cols(cls, n: int) -> "Type[Config]":
         ----------
         n
             number of columns to print
+        
+        Examples
+        --------
+>>> import polars as pl
```

Comment:
Could you indent this to the same level as the documentation? And maybe remove the latest two rows as they don't do anything?